public/home.html

```diff
@@ -8,11 +8,18 @@ <h1 class="mb-1 fw-semibold">
     </div>
   </div>
 </div>
+<script>
```

Comment:
It really errored for me when I didn't have it hoisted up there. Did you try it on your machine?

Reply:
Makes sense. I think it's because of the `type="module"` being scoped, so the global scope doesn't have it. I see you changed to using an event listener which I think is a better solution, anyway 👍